com.unity.netcode.gameobjects/Utilities/TestHelpers/Runtime/NetworkVariableHelper.cs

```diff
@@ -1,7 +1,7 @@
 using System;
 using System.Collections.Generic;
 
-namespace Unity.Netcode.RuntimeTests
+namespace Unity.Netcode.TestHelpers.Runtime
```

Comment:
I feel like we should just use `Unity.Netcode.TestHelpers` namespace for all and also drop `.Runtime`

Reply:
This was planning for any potential editor specific helpers and trying to keep it clear that these are runtime test only helper methods.
I can flatten it down, but it opens the possibility of someone adding editor specific helpers in this assembly (which would/could cause issues).